{
  "CVE Code": "CVE-2021-41200",
  "Published": "2021-11-05T20:00:12",
  "Updated": "2024-11-13T21:44:45.830Z",
  "Description": "TensorFlow is an open source platform for machine learning. In affected versions if `tf.summary.create_file_writer` is called with non-scalar arguments code crashes due to a `CHECK`-fail. The fix will be included in TensorFlow 2.7.0. We will also cherrypick this commit on TensorFlow 2.6.1, TensorFlow 2.5.2, and TensorFlow 2.4.4, as these are also affected and still in supported range.",
  "ProblemTypes": [
    "CWE-617: Reachable Assertion"
  ],
  "Affected": [
    {
      "vendor": "tensorflow",
      "product": "tensorflow",
      "versions": [
        ">= 2.6.0, < 2.6.1",
        ">= 2.5.0, < 2.5.2",
        "< 2.4.4"
      ]
    }
  ],
  "Reference": [
    "https://github.com/tensorflow/tensorflow/security/advisories/GHSA-gh8h-7j2j-qv4f",
    "https://github.com/tensorflow/tensorflow/issues/46909",
    "https://github.com/tensorflow/tensorflow/commit/874bda09e6702cd50bac90b453b50bcc65b2769e"
  ]
}